pkg/store/mv/c_list.go

```diff
@@ -0,0 +1,198 @@
+// Copyright 2019 ScyllaDB
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mv
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/gocql/gocql"
+)
+
+type List []Elem
+
+func (l List) ToString(colInfo gocql.TypeInfo) string {
+	out := "set<"
+	if colInfo.Type() == gocql.TypeList {
+		out = "list<"
+	}
+	if len(l) == 0 {
+		return out + ">"
+	}
+	listInfo := colInfo.(gocql.CollectionType)
+	for idx := range l {
+		out += fmt.Sprintf("%d:%s;", idx, l[idx].ToString(listInfo.Elem))
+	}
+	out = out[:len(out)-1]
+	return out + ">"
```

Comment:
Can you modify the last char to `>` in place instead?

Reply:
Can't. It's string, simple chars modifications impossible. Modified to `return out[:len(out)-1] + ">"`